backend/corpora/lambdas/api/v1/collection_id/upload.py

```diff
@@ -75,6 +75,10 @@ def upload_from_link(collection_id: str, token_info: dict, url: str, dataset_id:
     except NonExistentCollectionException:
         raise ForbiddenHTTPException()
     except InvalidProcessingStateException:
-        raise MethodNotAllowedException()
+        raise MethodNotAllowedException(
+            detail="Submission failed. A dataset cannot be updated while a previous update for the same dataset is in "
+            "progress. Please cancel the current submission with "
+            f"DELETE /v1/collections/collection/dataset/{dataset_id}"
```

Comment:
actually, should we specify the mechanism for canceling? Curators have more user-friendly notebooks they could use instead if they'd like (to more easily set headers and whatnot), or the UI could be available. What do you think of just saying please cancel the current submission?